dynamo/src/main/scala/d4s/codecs/D4SEncoder.scala

```diff
@@ -33,8 +35,8 @@ object D4SAttributeEncoder {
       AttributeValue.builder().l(ls.asJavaCollection).build()
   }
 
-  implicit def mapLikeEncoder[V, M[_, _] <: Map[String, V]](implicit V: D4SAttributeEncoder[V]): D4SAttributeEncoder[M[String, V]] = {
-    item: M[String, V] =>
+  implicit def simpleMapEncoder[V](implicit V: D4SAttributeEncoder[V]): D4SAttributeEncoder[Map[String, V]] = {
+    item: Map[String, V] =>
```

Comment:
I made this simple map encoder, 'cuz the previous implementation doesn't work.
Still, it's need to be implemented in a generic way.